doc/api/process.md

```diff
@@ -2309,6 +2309,227 @@ import { loadEnvFile } from 'node:process';
 loadEnvFile();
 ```
 
+### Environment variables file parsing specification
+
+This section describes how environment variables file parser reads a file
+and sets up the environment variables in Node.js.
```

Comment:
I think this can be reworded to:
This section describes how environment variables are read and parsed from a file.